blue-green-spring/README.md

```diff
@@ -0,0 +1,170 @@
+# A Sample OpenShift Pipeline for Blue Green deployments
+
+This example demonstrates how to implement a full end-to-end Jenkins Pipeline for a Java application in a Blue/Green deployment in the OpenShift Container Platform. This sample demonstrates the following capabilities:
+
+* Deploying an integrated Jenkins server inside of OpenShift
+* Running both custom and oob Jenkins slaves as pods in OpenShift
+* "One Click" instantiation of a Jenkins Pipeline using OpenShift's Jenkins Pipeline Strategy feature
+* Promotion of an application's container image within an OpenShift Cluster (using `oc tag`)
+* Tagging images with the current version of the artifact defined in the pom.xml file
+* Promotion of an application's container image to a blue/green production configuration
+* Switching production routes between blue and green deployments after confirmation
+
+## Quickstart
+
+Run the following commands to instantiate this example.
+
+```
+cd ./simple-spring-boot
```

Comment:
If you run the pipeline a second time it will switch to blue.  Each time it is ran the route switches from the service to service.  I probably need to create two more routes one that always points to blue and one that always points to green.  This way the new release can be checked out before the switch over.

Reply:
@malacourse yeah, and I think a little more explanation of how the flow will work is needed in the README.

Reply:
I have updated the Pipeline with readme changes and added two more routes.  Review when you get a chance.

Reply:
@malacourse the directory name here still doesn't match the directory you're actually creating